txbits/app/models/UserModel.scala

```diff
@@ -57,7 +57,7 @@ class UserModel(val db: String = "default") {
   }
 
   def addFakeMoney(uid: Long, currency: String, amount: BigDecimal) = DB.withConnection(db) { implicit c =>
-    if (Play.current.configuration.getBoolean("fakeexchange").get) {
+    if (globals.fakeExchange) {
```

Comment:
There is no disadvantage to looking up configs each time. They are all cached anyway.